<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoWindow</class>
 <widget class="QWidget" name="DemoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Zoom Level:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="zoom_spin"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>148</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Map Source:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="map_source_combo"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="GeoQt::MapWidget" name="map_widget"/>
   </item>
   <item>
    <widget class="QLabel" name="coordinate_label">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GeoQt::MapWidget</class>
   <extends>QGraphicsView</extends>
   <header>MapWidget</header>
   <slots>
    <signal>mapCenterChanged(QPointF)</signal>
    <signal>zoomLevelChanged(int)</signal>
    <slot>setMapCenter(QPointF)</slot>
    <slot>setZoomLevel(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>zoom_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>map_widget</receiver>
   <slot>setZoomLevel(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>map_widget</sender>
   <signal>zoomLevelChanged(int)</signal>
   <receiver>zoom_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
